### ISSUE TITLE
module code checks report possible issues

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/Biomass_regeneration#17]
- [ ] Biomass_regeneration: module code: cohortData, inactivePixelIndex, pixelGroupMap, rstCurrentBurn are declared in metadata inputObjects, but no default(s) are provided in .inputObjects
- [ ] Biomass_regeneration: inputObjects: rasterToMatch is used from sim inside FireDisturbance, but is not declared in metadata inputObjects

**Comment by CeresBarros** [comment]
> Biomass_regeneration: module code: cohortData, inactivePixelIndex, pixelGroupMap, rstCurrentBurn are declared in metadata inputObjects, but no default(s) are provided in .inputObjects
 
This was relatively deliberate. We had decided that this model is not supposed to be run stand-alone. However, we can of course review this decision and add defaults. I think it's a decision that depends on "how much we always want to be able to run _any_ in stand-alone mode". Before we were okay with some modules (the regeneation ones) not being meant to run stand alone, maybe now we're not?
In any case, I'm totally fine with adding defaults, but this is not high up in my priority list at the moment :/

> Biomass_regeneration: inputObjects: rasterToMatch is used from sim inside FireDisturbance, but is not declared in metadata inputObjects

nice catch. that should be added ASAP

**Comment by achubaty** [comment]
I agree that the first set of these are probably OK to ignore but I'd rather have a proper mechanism to silence these messages (otherwise we risk us + users ignoring messages from here and other modules in the future).

**Comment by CeresBarros** [comment]
I don't agree. To me it's similar to asking `lme4` developpers to silence/make a decision about model singularity warnings. Sure many people ignore them (bad practice) or panic (newbies). But people that take time to learn, will find out when to worry about it and when they can be "ignored".
Silencing code checks is us making a ubiquitous decision about what is/is not okay for every module. I think the conservatism of the code checks is beneficial during development. I also think that users can learn from them and learn to turn them off. SpaDES needs training and this is part of the training. What we really need is good vignettes, quick lessons, etc. Now that I think of it, this should be part of [SpaDES4Dummies](https://ceresbarros.github.io/SpaDES4Dummies/)
